android/src/com/mapswithme/maps/widget/placepage/SponsoredHotel.java

```diff
@@ -73,6 +173,15 @@ static void requestPrice(String id, String currencyCode)
     nativeRequestPrice(id, currencyCode);
   }
 
+  static void requestInfo(String id, String locale)
+  {
+    HotelInfo info = sInfoCache.get(id);
+    if (info != null)
+      onInfoReceived(id, info);
+
+    nativeRequestInfo(id, locale);
```

Comment:
onInfoReceived() also called from this:
https://github.com/goblinr/omim/blob/MAPSME-1-external-place-page-booking/android/jni/com/mapswithme/maps/SponsoredHotel.cpp#L154